dev/interrupt/arm_gic/arm_gic.c

```diff
@@ -219,7 +219,8 @@ static void arm_gic_resume_cpu(uint level)
         dprintf(SPEW, "%s: distibutor is off, calling arm_gic_init instead\n", __func__);
         arm_gic_init();
         resume_gicd = true;
-    } else {
```

Comment:
nit: leave this the way it was

Reply:
sure. this happened once I ran the codestyle.space script. I will revert.